src/Components/LandingPage/LandingPage.tsx

```diff
@@ -123,29 +133,31 @@ const LandingPage = ({ handleCheckboxChange }: LandingPageProps) => {
       <h1 className="sub-header">
         <FormattedMessage id="disclaimer.header" defaultMessage="What you should know: " />
       </h1>
-      <Card variant="outlined">
+      <Card variant="outlined" sx={{ backgroundColor: theme.lightPurpleBackground }}>
         <CardContent>
           <Typography variant="body1">
-            <FormattedMessage
-              id="landingPage.body"
-              defaultMessage="MyFriendBen is a tool that can help determine benefits you are likely eligible for. Here's what you should know before you get started:"
-            />
+            <div className="main-paragraph">
+              <FormattedMessage
+                id="landingPage.body"
+                defaultMessage="MyFriendBen is a tool that can help determine benefits you are likely eligible for. Here's what you should know before you get started:"
+              />
+            </div>
           </Typography>
           <ul className="landing-page-list-container">
-            <li>
+            <li className="main-paragraph">
               <FormattedMessage
                 id="landingPage.firstBulletItem"
                 defaultMessage="MyFriendBen only provides estimates of what you may qualify for. You should not rely on these estimates. You must confirm your final eligibility and benefit amount with the proper agency or other decision maker."
               />
             </li>
             <li>
-              <p>
+              <p className="main-paragraph">
                 <FormattedMessage
                   id="landingPage.publicCharge"
                   defaultMessage="Some benefits are available to Non-U.S. citizens. Non-U.S. citizens planning to apply for legal permanent residency or a visa should consider how applying for any benefits may affect their immigration status. For more information, please review the "
                 />
                 <a
-                  className="public-charge-link"
+                  style={{ color: '#B85A27' }}
```

Comment:
I think you forgot one.
```suggestion
                  style={{ color: theme.terraCottaColor }}
```